src/zenml/cli/stack_components.py

```diff
@@ -1362,6 +1380,988 @@ def flavor_group() -> None:
     )(explain_command)
 
 
+def register_secrets_manager_subcommands() -> None:
```

Comment:
This file is already as bloated as it gets. Do you think you could keep these new `register_*` functions inside their original files, secret.py and served-models.py (perhaps renamed to model.py) instead ?

That should give some much needed structure to this already convoluted sub-command logic. Not to mention it should make the review easier :)

Reply:
Yes of course. I'll do that.

Reply:
Done